modules/@apostrophecms/asset/index.js

```diff
@@ -116,14 +117,26 @@ module.exports = {
       build: {
         usage: 'Build Apostrophe frontend CSS and JS bundles',
         afterModuleInit: true,
-        async task(argv) {
+        async task(argv = {}) {
           // The lock could become huge, cache it, see computeCacheMeta()
           let packageLockContentCached;
           const req = self.apos.task.getReq();
           const namespace = self.getNamespace();
           const buildDir = `${self.apos.rootDir}/apos-build/${namespace}`;
           const bundleDir = `${self.apos.rootDir}/public/apos-frontend/${namespace}`;
           const modulesToInstantiate = self.apos.modulesToBeInstantiated();
+          // Make it clear if builds should detect changes
+          const detectChanges = Array.isArray(argv.changes);
+          // Remove invalid changes. `argv.changes` is an array of relative
```

Comment:
It bothers me that a command line task would have options that aren't really usable on the command line (there's no way to pass an array). It's not a blocker for now though.